src/components/StudioCard/StudioCard.js

```diff
@@ -1,7 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import {
+  Card,
+  ImgBox,
+  Content,
+  ContentPrice,
+  Title,
+  StyledSlider,
+  StyledLink,
+  WishBtn,
+  OutlineHeart,
+  FillHeart,
+} from './StyleStudioCard';
```

Comment:
```suggestion
import * as S from './StyleStudioCard';
```
점점 컴포넌트가 많아질 수록 길어지게 되면 가독성에 영향을 줄 수 있습니다.
이럴 경우에는 위와 같은 방식으로 줄일 수 있습니다!